package.json

```diff
@@ -30,8 +30,10 @@
     "@fortawesome/free-brands-svg-icons": "^5.10.2",
     "@fortawesome/free-solid-svg-icons": "^5.10.2",
     "@ng-select/ng-select": "^3.7.3",
+    "@nguniversal/express-engine": "^9.1.1",
```

Comment:
@Nishanthh Is this required?

Reply:
@abheek yes bro, dependency module in runtime